src/Entity/MembershipTermEntity.php

```diff
@@ -8,53 +8,57 @@
 use Drupal\Core\Entity\EntityChangedTrait;
 use Drupal\Core\Entity\EntityTypeInterface;
 use Drupal\membership\EventDispatcherTrait;
-use Drupal\membership\MembershipOfferInterface;
 use Drupal\user\UserInterface;
 
 /**
- * Defines the Membership Offer entity.
+ * Defines the Membership term entity entity.
  *
  * @ingroup membership
  *
  * @ContentEntityType(
- *   id = "membership_offer",
- *   label = @Translation("Membership Offer"),
+ *   id = "membership_term_entity",
```

Comment:
Is there a reason for the `_entity` portion? Most D8 entities just define themselves with a meaningful machine name, I think the `_entity` piece is a little duplicative and implies an additional distinction where there is none?